logo.svg

```diff
@@ -0,0 +1,26 @@
+<svg
```

Comment:
So this is an SVG file that is merely a wrapper for a PNG file?  What is this for?